pkgs/development/libraries/mesa/default.nix

```diff
@@ -118,11 +112,9 @@ self = stdenv.mkDerivation {
     "-Ddri-search-path=${libglvnd.driverLink}/lib/dri"
 
     "-Dplatforms=${concatStringsSep "," eglPlatforms}"
-    "-Ddri-drivers=${concatStringsSep "," driDrivers}"
     "-Dgallium-drivers=${concatStringsSep "," galliumDrivers}"
     "-Dvulkan-drivers=${concatStringsSep "," vulkanDrivers}"
 
-    "-Ddri-drivers-path=${placeholder "drivers"}/lib/dri"
```

Comment:
Leaving this in seems to avoid the referency cycle.

Reply:
Thanks :) I missed some important references when I did a quick grep through the source-code :o